src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -768,6 +769,18 @@ export class StudyViewPageStore {
             }, [] as MutationGeneFilter[]);
         }
 
+        if (_.isArray(filters.fusionGenes) && filters.fusionGenes.length > 0) {
+            this._fusionGeneFilter = _.reduce(filters.fusionGenes, (acc, next) => {
```

Comment:
i can see that this logic follows pattern of existing blocks in use of _.reduce, but worth noting that these reduces are really mapping operations.  would be more semantic to use map.   reduce should only be used if there's a conditional inclusion (filter would also work) or if we're producing a map from an array.     also, we can now use native reduce, map, filter methods of array object.  lodash unnecessary in these cases.

Reply:
@alisman Won't the map alter the original array?

Reply:
Sorry, it was for another scenario. In this case, it will not. Will update. Thanks.